Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'turbolinks'
 
 # View
 gem 'simple_form'
+gem 'enumerize'
```

Comment:
@cawel I'm understand your confusion. I thought this had been rebased with `master` recently. I'm not sure the last time you rebased, but when we merged PR the `enumerize` gem will appear twice in the Gemfile. Please view below:

1. https://github.com/cawel/Montreal.rb/blob/pages-cleanup/Gemfile#L29
2. https://github.com/montrealrb/Montreal.rb/blob/master/Gemfile#L23

My advice is to rebase again, please. Then the Gemfile in your branch should not include an entry for `enumerize`.